pull_request_template.md

```diff
@@ -1,6 +1,11 @@
 ## Purpose
 > Describe the problems, issues, or needs driving this feature/fix and include links to related issues in the following format: Resolves issue1, issue2, etc.
 
+$subject

```

Comment:
We encourage the users to add a detailed description of the pull request here. Hence is it appropriate to add `$subject` here?

Reply:
I think even if we don't add `$subject` by default, devs tend to type `$subject` and submit PRs, and for most PRs, $subject is sufficient as the PR title is self explanatory.

In case PR description is felt incomplete/insufficient, IMO it should be the responsibility of the reviewer to point that out.

Reply:
I don't see the `$subject` in the oxygen repo [pull request template](https://github.com/wso2/oxygen-ui/blob/main/pull_request_template.md). However, I noticed that it has been newly added to the identity-apps repo [pull request template](https://github.com/wso2/identity-apps/blob/master/pull_request_template.md) @Achintha444.

I feel that we encourage users to add `$subject` here rather than a detailed description, it might be better to leave the existing description for its intended purpose and allow users to add `$subject` only if there's a specific requirement. Let's get the others' input as well. :)